src/apis/message/queries.ts

```diff
@@ -0,0 +1,40 @@
+import { useMutation, useQuery } from '@tanstack/react-query'
+import {
+  getMessageRooms,
+  getMessage,
+  createMessage,
+  deleteMessageRoom
+} from './apis'
+import type {
+  CreateMessageReq,
+  GetMessageReq,
+  GetMessageRoomsReq
+} from './types'
+
+export const useGetMessageRooms = (params: GetMessageRoomsReq) =>
+  useQuery({
+    queryKey: ['getMessageRooms'],
```

Comment:
```suggestion
    queryKey: ['getMessageRooms', params],
```
params가 변경되면 자동으로 refetch되어야하지 않나요?!
따로 상태가 변경되었을 때, refetch되는걸까요??